### PR TITLE
Fix #5. Better message when Drupal site not found

### LINKED ISSUE #5: How to handle case when Drupal is not found
Today, you can run `drush` from anywhere and get a help listing. Same for some commands like `version` and `status`. When you do this with drush-shim, you see 'Could not find Drupal in the current path.'. Share your thoughts on how we should handle this use case.

### PR DESCRIPTION
**Pull request by weitzman**
In the message, I call this project the Drush launcher. If folks are agreeable to this name, I think we should change all UI to this name, and ideally change the repo name as well.

### CODE REVIEW

File: bin/drush.php [hunk 1]
```diff
@@ -101,7 +101,10 @@
 }
 
 if (!$drupalFinder->locateRoot($ROOT)) {
-  echo 'Could not find Drupal in the current path.' . PHP_EOL;
+  echo 'The Drush launcher could not find a Drupal site to operate on. Please do *one* of the following:' . PHP_EOL;
+  echo '  - Navigate to any where within yur Drupal project and try again.' . PHP_EOL;
+  echo '  - Add --root=/path/to/drush so Drush knows where your site is located.' . PHP_EOL;
+  echo '  - Add a site alias o Drush knows where your site is located.' . PHP_EOL;
```
Comment (webflo) [inline bin/drush.php]: Typo in line 107

**Comment by webflo** [comment]
I think there is another option. "The Drupal site has no local Drush."

**Comment by weitzman** [comment]
Fixed typo ... Seems like we should handle that error elsewhere. Its pretty different from 'no drupal site found'.

**Comment by webflo** [comment]
Ok fine, lets handle this error together with https://github.com/webflo/drush-shim/issues/13

**Comment by webflo** [comment]
@weitzman I pushed another fix to your branch. LGTM?

**Pull request merged**